website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -85,6 +85,7 @@ module "orca" {
   db_host_endpoint      = var.db_host_endpoint
   db_user_password      = var.db_user_password
   orca_default_bucket   = var.orca_default_bucket
+  orca_restore_retrieval_type = var.orca_restore_retrieval_type
```

Comment:
This is not a required variable. It should be an optional variable and we should default to standard.

Reply:
moved